02-microarray/pathway-analysis_microarray_03_gsva.Rmd

```diff
@@ -347,11 +347,14 @@ mapped_df %>%
 
 We can see that the associated values vary for each row.
 
-As we mentioned earlier, we will want to remove duplicated gene identifiers in preparation for the GSVA steps later, so let's keep the Entrez IDs associated with the maximum expression value per sample.
-Note that using the maximum expression value to collapse duplicate gene identifiers is just one of the expression values that can be used here.
-See the [Gene Set Enrichment Analysis (GSEA) User guide](https://www.gsea-msigdb.org/gsea/doc/GSEAUserGuideFrame.html) for more information on other expression values that can be used [@GSEA-user-guide]. 
+As we mentioned earlier, we will want to collapse any duplicate gene identifiers in preparation for the GSVA steps later.
+GSVA is executed on a per sample basis so let's keep the maximum expression value per sample associated with the duplicate gene identifiers.
```

Comment:
```suggestion
GSVA is executed on a per sample basis so let's keep the maximum expression value per sample associated with the duplicate Ensembl gene identifiers.
```

When there's multiple gene id types, it can help to remind us which one is the duplicate here that we are talking about.

Reply:
Actually, I believe the Entrez ID is the duplicate here, multiple Ensembl gene identifiers mapped to the same Entrez ID.

Reply:
Ah right. I got confused too. We’ll all the more reason to specify.